JetStreamDriver.js

```diff
@@ -2359,21 +2324,51 @@ function processTestList(testList)
     for (let name of benchmarkNames) {
         name = name.toLowerCase();
         if (benchmarksByTag.has(name))
-            globalThis.JetStream.enableBenchmarksByTag(name);
+            benchmarks.push(...enableBenchmarksByTag(name));
         else
-            globalThis.JetStream.enableBenchmarksByName(name);
+            benchmarks.push(enableBenchmarkByName(name));
     }
+    return benchmarks;
+}
+
+
+function enableBenchmarkByName(name) {
+    const benchmark = benchmarksByName.get(name.toLowerCase());
+
+    if (!benchmark)
+        throw new Error(`Couldn't find benchmark named "${name}"`);
+
+    return benchmark;
 }
 
+
+function enableBenchmarksByTag(tag, excludeTags) {
+    let benchmarks = benchmarksByTag.get(tag.toLowerCase());
+    if (!benchmarks) {
+        const validTags = Array.from(benchmarksByTag.keys()).join(", ");
+        throw new Error(`Couldn't find tag named: ${tag}.\n Choices are ${validTags}`);
+    }
+    if (excludeTags) {
+        benchmarks = benchmarks.filter(benchmark => {
+            return !benchmark.hasAnyTag(...excludeTags);
+        });
+    }
+    return benchmarks;
+}
+
+
+let benchmarks = [];
 const defaultDisabledTags = [];
 // FIXME: add better support to run Worker tests in shells.
 if (!isInBrowser)
     defaultDisabledTags.push("WorkerTests");
 
 if (typeof testList !== "undefined") {
-    processTestList(testList);
+    benchmarks = processTestList(testList);
 } else if (customTestList.length) {
-    processTestList(customTestList);
+    benchmarks = processTestList(customTestList);
```

Comment:
possible future nit / clarity question: What's the difference between `testList` and `customTestList`. Could we get rid of the latter, if they basically serve the same purpose?